Gemfile.lock

```diff
@@ -0,0 +1,64 @@
+PATH
+  remote: .
+  specs:
+    sensu-plugins-chrony (0.0.10)
+      sensu-plugin (= 1.2.0)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    ast (2.3.0)
+    diff-lcs (1.3)
+    github-markup (1.6.1)
+    json (2.1.0)
+    mixlib-cli (1.7.0)
+    parallel (1.12.0)
+    parser (2.4.0.0)
+      ast (~> 2.2)
+    powerpack (0.1.1)
+    rainbow (2.2.2)
+      rake
+    rake (12.2.1)
+    redcarpet (3.4.0)
+    rspec (3.7.0)
+      rspec-core (~> 3.7.0)
+      rspec-expectations (~> 3.7.0)
+      rspec-mocks (~> 3.7.0)
+    rspec-core (3.7.0)
+      rspec-support (~> 3.7.0)
+    rspec-expectations (3.7.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.7.0)
+    rspec-mocks (3.7.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.7.0)
+    rspec-support (3.7.0)
+    rubocop (0.49.1)
+      parallel (~> 1.10)
+      parser (>= 2.3.3.1, < 3.0)
+      powerpack (~> 0.1)
+      rainbow (>= 1.99.1, < 3.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (~> 1.0, >= 1.0.1)
+    ruby-progressbar (1.9.0)
+    sensu-plugin (1.2.0)
+      json
+      mixlib-cli (>= 1.1.0)
+    unicode-display_width (1.3.0)
+    yard (0.9.9)
+
+PLATFORMS
+  ruby
+
+DEPENDENCIES
+  bundler (~> 1.7)
+  github-markup (~> 1.3)
+  rake (~> 12.0)
+  redcarpet (~> 3.2)
+  rspec (~> 3.4)
+  rubocop (~> 0.49.0)
+  sensu-plugins-chrony!
+  yard (~> 0.8)
+
+BUNDLED WITH
+   1.16.0
```

Comment:
Remove lock file. Looks like we just need to add our standard .gitignore as well.

Reply:
good call